### PR TITLE
api calls to the server and created productContext

### PR DESCRIPTION
**Pull request by pariyar07**
Screenshot 2022-03-28 at 5 59 50 PM

### CODE REVIEW

**Review by yogesh- — APPROVED** [review]
- Files structure is good
- code is readable and in smaller chunks
- UI is decent.

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=bjbjboaekcnelcn131wbc8wgcbuwcvwu766
```
Comment (viru2001) [inline .env:1]: you should not push env file

**Review by viru2001 — APPROVED** [review]

File: .env (hunk 1 above)
Comment (viru2001) [inline .env]: you should not push env file on GitHub

File: src/components/Home/common/product-listing-card.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    <div className="card-title">
+                        <div>
+                            <h3>{name}</h3>
+                            <p className="card-description">{info}</p>
+                        </div>
+                    </div>
+                    <div className="price">
+                        <p className="current-price">${price}</p>
+                        <p className="actual-price">{actualPrice}</p>
+                        <p className="discount-percentage">(50% OFF)</p>
```
Comment (viru2001) [inline src/components/Home/common/product-listing-card.jsx]: you can discount the percentage in the database

File: src/contexts/product-context.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const ProductsContext = createContext(null)
+
+const ProductsProvider = ({ children }) => {
+    const [products, setProducts] = useState([]);
+
+    useEffect(() => {
+        (async () => {
+            try {
+                const productData = await axios.get("/api/products");
+                return setProducts(productData.data.products)
```
Comment (viru2001) [inline src/contexts/product-context.jsx]: why you used return here ?

Comment (pariyar07) [reply src/contexts/product-context.jsx]: thats by mistake

File: src/components/Home/common/product-listing-card.jsx (hunk 2 above)
Comment (pariyar07) [reply src/components/Home/common/product-listing-card.jsx]: Done

File: .env (hunk 1 above)
Comment (pariyar07) [reply .env]: Yup thanks for pointing out.

**Pull request merged**